met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx

```diff
@@ -60,7 +59,7 @@ const CreateFolderForm = () => {
     };
 
     const handleFolderNameChange = (name: string) => {
-        validate();
+        isValid();
```

Comment:
does it do anything ?
we call a boolean and dont do anything.is that ok?

Reply:
but it doesnt return anything and doesnt throw an error?

Reply:
no error but i think it returns a boolean

Reply:
hm.. how does the boolean gets used here?
dont we need to return ?

Reply:
okay; i think if isValid is false ; setFolderName(name) wont be invoked?